README.md

```diff
@@ -6,295 +6,354 @@
+    ```bash
+    ibmcloud target --cf -g <default>
+    ```
+6.  Set the Kubernetes context of your CLI to your cluster so that you can run `kubectl` commands. For example, if your cluster is in IBM Cloud Kubernetes Service, run the following command.
+    ```bash
+    ibmcloud ks cluster config -c <cluster_name_or_ID>
+    ```
 
-## Set up the operator
+    To check that your Kubernetes context is set to your cluster, run the following command.
+    ```bash
+    kubectl config current-context
+    ```
 
-To set up the operator after logging in with `ibmcloud`, run the below installer.
+[Back to top](#ibm-cloud-operator)
 
-By default, the script will create a new API key for use in the operator. To use a custom API key, set the `IBMCLOUD_API_KEY` environment variable to the key.
+## Setting up the operator
 
-#### Using a ServiceId
+You can use an installation script to set up the IBM Cloud Operator. The installation script stores an API key in a Kubernetes secret in your cluster that can be accessed by the IBM Cloud Operator. Next, the script sets default values that are used to provision IBM Cloud services, like the resource group and region to provision the services in. Later, you can override any default value in the `Service` custom resource. Finally, the script deploys the operator in your cluster in the `ibmcloud-operators` namespace.
 
-To instantiate services and bindings on behalf of a ServiceId, set the environment variable `IBMCLOUD_API_KEY` to the `api-key` of the ServiceId. This can be obtained via the IBM Cloud Console or [CLI](https://cloud.ibm.com/docs/iam?topic=iam-serviceids). Be sure to give proper access permissions to the ServiceId.
+Prefer to create the secrets and defaults yourself? See the [IBM Cloud Operator installation guide](docs/install.md).
 
-Next log into the IBM Cloud account that owns the ServiceId and follow the instructions above.
-
-<!-- END SHOW operator hub -->
+### Using a service ID
 
-### Install
+By default, the installation script creates an IBM Cloud API key that impersonates your user credentials, to use to set up the IBM Cloud Operator. However, you might want to create a service ID in IBM Cloud Identity and Access Managment (IAM). By using a service ID, you can control access for the IBM Cloud Operator without having the permissions tied to a particular user, such as if that user leaves the company. For more information, see the [IBM Cloud docs](https://cloud.ibm.com/docs/account?topic=account-serviceids).
 
-To install the latest stable release of the operator, run the below script.
+1.  Create a service ID in IBM Cloud IAM. 
+    ```bash
+    ibmcloud iam service-id-create serviceid-ico -d service-ID-for-ibm-cloud-operator
+    ```
+2.  Assign the service ID access to the required permissions.<!--what are these? maybe just administrator? or should they scope to a different role, different service, region, resource group, etc?-->
```

Comment:
What are the required permissions for the service ID? maybe just administrator? or should they scope to a different role, different service, region, resource group, etc?

Reply:
Great question. They need to be able to provision services with this access, so presumably some kind of administrator would be needed. Basically everything you need the operator to do (e.g. create resources, delete them, create bindings, etc) it needs to have access to. It can be as narrow or as wide a net as needed.